platform/cosmos/api.go

```diff
@@ -11,16 +12,21 @@ import (
 )
 
 type Platform struct {
-	client Client
+	client    Client
+	CoinIndex uint
 }
 
 func (p *Platform) Init() error {
-	p.client = Client{blockatlas.InitClient(viper.GetString("cosmos.api"))}
+	p.client = Client{blockatlas.InitClient(viper.GetString(p.ConfigKey()))}
```

Comment:
Same pattern as btc, created an issue to improve this: https://github.com/trustwallet/blockatlas/issues/668